manifests/j/JHubi1/OllamaApp/1.2.0/JHubi1.OllamaApp.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: JHubi1.OllamaApp
+PackageVersion: 1.2.0
+InstallerType: inno
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/JHubi1/ollama-app/releases/download/1.2.0/ollama-windows-x64-v1.2.0.exe
+  InstallerSha256: 40CBEAC86BF015E46E7353E8D5BD02140C9503882EC4670B92D33D519F1FA460
+  InstallerSwitches:
+    Custom: "/CURRENTUSER /SUPPRESSMSGBOXES"

```

```suggestion
  InstallerSwitches:
    Custom: "/CURRENTUSER /SUPPRESSMSGBOXES"
  Scope: user
```
If the installer also supports machine scope, it would be best to add another installer node with the `/ALLUSERS` switch